### PR TITLE
feat:Added user on boarding page

### PR DESCRIPTION
**Pull request by tushargoyal812**
extension-onbparding

### CODE REVIEW

File: chrome-extension/src/App.css [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  background-color: white;
+  height: 12rem;
+  width: 15rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  align-items: center;
+}
+
+/* .todo-input{
```
Comment (deepak29-git) [inline chrome-extension/src/App.css:44]: remove commented code

File: chrome-extension/src/App.css [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  align-self: center;
+  justify-self: center;
+} */
+
+.main-heading{
+  color: white;
```
Comment (deepak29-git) [inline chrome-extension/src/App.css:50]: use color variable

File: chrome-extension/src/App.js [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+function App() {  
+
+  const [userName,setUserName]=useState()
+  const [userEmail,setUserEmail]=useState("")
+  const [isEmail,setIsEmail]=useState(false)
+  const [showName,setShowName]=useState(true)
+  const [isPassword,setIsPassword]=useState(false)
+  const [Time,setTime]=useState()
+  const [focus,setFocus]=useState(false)
+  const [greeting,setGreeting]=useState()
```
Comment (sharmayatendra) [inline chrome-extension/src/App.js:14]: you can use reducer over here since states are quite more in number

**Review by sharmayatendra — APPROVED** [review]
nothing major caught my eye. looks good to me
left a suggestion can look upon it

File: chrome-extension/src/App.js [hunk 4]
```diff
@@ -0,0 +1,88 @@
+import { useState } from 'react';
+import './App.css';
+import {v4 as uuid} from "uuid";
+
+function App() {  
+
+  const [userName,setUserName]=useState()
+  const [userEmail,setUserEmail]=useState("")
+  const [isEmail,setIsEmail]=useState(false)
+  const [showName,setShowName]=useState(true)
```
Comment (deepak29-git) [inline chrome-extension/src/App.js:10]: you can reduce the number of usestate by using object

**Review by deepak29-git — APPROVED** [review]
liked : your variable name is good, 
you can remove react logo

File: chrome-extension/src/App.js (hunk 3 above)
Comment (tushargoyal812) [reply chrome-extension/src/App.js:14]: yes will do it in final version.

File: chrome-extension/src/App.css (hunk 1 above)
Comment (tushargoyal812) [reply chrome-extension/src/App.css:44]: ok

File: chrome-extension/src/App.css (hunk 2 above)
Comment (tushargoyal812) [reply chrome-extension/src/App.css:50]: yes will do it

File: chrome-extension/src/App.js (hunk 4 above)
Comment (tushargoyal812) [reply chrome-extension/src/App.js:10]: ok

**Pull request merged**